qemu/tests/cfg/tpm_verify_device.cfg

```diff
@@ -31,13 +31,13 @@
         pattern_output_get_tpm = TpmPresent\s+:\s+True;TpmReady\s+:\s+True
     variants:
         - with_emulator:
-            tpm_type_tpm0 = emulator
-            tpm_model_tpm0 = tpm-crb
+            tpm_type = emulator
+            tpm_model = tpm-crb
             ppc64le, ppc64:
-                tpm_model_tpm0 = tpm-spapr
+                tpm_model = tpm-spapr
             aarch64:
-                tpm_model_tpm0 = tpm-tis-device
-            tpm_version_tpm0 = 2.0
+                tpm_model = tpm-tis-device
+            tpm_version = 2.0
```

Comment:
Seems we have a dup tpm_version here ?

Reply:
Yes, it's a history issue, I will delete it.

Reply:
Thanks for fixing the history issue.

Others LGTM.

 (2/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.q35: STARTED
 (2/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.q35: PASS (95.04 s)
 (3/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_check_ovmf.q35: STARTED
 (3/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_check_ovmf.q35: PASS (94.87 s)
 (4/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_reboot.q35: STARTED
 (4/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_reboot.q35: PASS (903.66 s)
 (5/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_multi_vms.q35: STARTED
 (5/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_multi_vms.q35: PASS (222.42 s)